### ISSUE TITLE
Adding emotion libary to example starter dependencies so the project can run 

### ISSUE THREAD

**Issue by WaterlessPiano2** [solana-labs/wallet-adapter#372]
**Describe the bug**
Adding emotion libary to example starter dependencies so the project can run 

**To Reproduce**
[Clone, install and run (dev) the started project](https://github.com/solana-labs/wallet-adapter/tree/master/packages/starter/example). When you navigate to loaclhos:3000 you will see the above error.

**Expected behavior**
The project should run and I can use it on my browser.

**Screenshots**
```
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
error - ./node_modules/@mui/styled-engine/index.js:6:0
Module not found: Can't resolve '@emotion/styled'

Import trace for requested module:
./node_modules/@mui/system/esm/index.js
./node_modules/@mui/material/styles/index.js
./node_modules/@mui/material/index.js
./components/ContextProvider.tsx
./pages/_app.tsx

https://nextjs.org/docs/messages/module-not-found
wait  - compiling /_error (client and server)...
```

**Desktop (please complete the following information):**
 - OS: ubuntu
 - Browser chrome
 - Version Version 99.0.4844.74 (Official Build) (64-bit)

**Additional context**
[The Solution to this problem and more context is in this pull Request
](https://github.com/solana-labs/wallet-adapter/pull/371)

**Comment by WaterlessPiano2** [comment]
https://github.com/solana-labs/wallet-adapter/pull/371

**Comment by jordansexton** [comment]
Closed by #371
